src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -1,12 +1,46 @@
 from py42.sdk.queries.fileevents.file_event_query import FileEventFilterStringField
+from py42.sdk.queries.query_filter import filter_attributes
+from py42.sdk.queries.query_filter import QueryFilterBooleanField
 from py42.sdk.queries.query_filter import QueryFilterTimestampField
 
 
 class EventTimestamp(QueryFilterTimestampField):
-    """Class that filters events based on the timestamp of the event that occurred."""
+    """Class that filters events based on the timestamp of the event that occurred.
+
+    This class has to create instance only with class method `within_the_last`.
```

Comment:
Probably good idea to add @annie-payseur as a reviewer

Reply:
I think this comment is inaccurate, you can use the other method but these _constants_ can only be used with `within_the_last`.

Reply:
Changed.